tests/rkt_net_test.go

```diff
@@ -116,6 +117,52 @@ func TestNetHostConnectivity(t *testing.T) {
 }
 
 /*
+ * None networking
+ * ---
+ * must be in an empty netns
+ */
+func TestNetNone(t *testing.T) {
+	testImageArgs := []string{"--exec=/inspect --print-netns --print-iface-count"}
+	testImage := patchTestACI("rkt-inspect-networking.aci", testImageArgs...)
+	defer os.Remove(testImage)
+
+	ctx := newRktRunCtx()
+	defer ctx.cleanup()
+
+	cmd := fmt.Sprintf("%s --debug --insecure-skip-verify run --net=none --mds-register=false %s", ctx.cmd(), testImage)
```

Comment:
Should we remove `--mds-register=false`?

Maybe from every test? (It doesn't have to be in this PR)